extensions/arc/src/ui/dashboards/postgres/postgresParameters.ts

```diff
@@ -221,9 +220,10 @@ export abstract class PostgresParametersPage extends DashboardPage {
 							cancellable: false
 						},
 						async (_progress, _token): Promise<void> => {
-							const session = await this._postgresModel.controllerModel.acquireAzdataSession();
+							let session: azdataExt.AzdataSession | undefined = undefined;
 							try {
-								this.resetAllParameters(session);
+								session = await this._postgresModel.controllerModel.acquireAzdataSession();
```

Comment:
Same here - you shouldn't need to move the session up. Just awaiting the resetAllParameters call should be enough.